doc/conf.py

```diff
@@ -46,6 +46,8 @@
     'exclude-members': '__weakref__'
 }
 
```

Comment:
Adding `autodoc_typehints = 'description'` to `conf.py` tells Sphinx (the documentation generator) to not include type hints in the function headers when it generates a new readthedocs website. For example, take a look at [this](https://pykoop.readthedocs.io/en/stable/_autosummary/pykoop.lmi_regressors.LmiHinfZpkMeta.html#pykoop.lmi_regressors.LmiHinfZpkMeta):

```python
 class LmiHinfZpkMeta(hinf_regressor: Optional[KoopmanRegressor] = None, type: str = 'post', zeros: Optional[Union[float, ndarray]] = None, poles: Optional[Union[float, ndarray]] = None, gain: float = 1, discretization: str = 'bilinear', t_step: float = 1, units: str = 'rad/s')
```

 It's way too much info at once, and it's repeated later on anyway. With this change it would look like this:

```python
class LmiHinfZpkMeta(hinf_regressor=None, type='post', zeros=None, poles=None, gain=1, discretization='bilinear', t_step=1, units='rad/s')
```